server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/MultipleDatabaseChangeRuntimeFlowableTask.java

```diff
@@ -250,6 +247,7 @@ protected void onSuccessful(Long taskId, TaskService taskService) {
         } finally {
             MultipleDatabaseChangeTraceContextHolder.clear();
         }
+        super.onSuccessful(taskId, taskService);
```

Comment:
As above, it is recommended to keep the original location

Reply:
ok, I moved it to the original location~